tools/submission/submission_checker.py

```diff
@@ -1226,7 +1226,7 @@ def check_performance_dir(
     # Check if this run uses early stopping. If it does, get the
     # min_queries from the detail log, otherwise get this value
     # from the config
-    if not uses_early_stopping:
+    if not (uses_early_stopping or (equal_issue_used_check and config.requires_equal_issue(model, division))):
         required_min_query_count = config.get_min_query_count(model, scenario)
```

Comment:
The problem can be solved if we read the min_query_count from the loadgen log and not the config right?

Reply:
Mmmm I don't think so. We are currently comparing the `min_query_count` from the log and with a fixed value in the config

Reply:
The comparison should be between the value from the log and the value stored in submission checker right?

Reply:
@pgmpablo157321 `min_query_count` is coming from the [loadgen log](https://github.com/pgmpablo157321/inference/blob/min_query_count_check/tools/submission/submission_checker.py#L1121). So, if equal issue mode is enabled, this key value should be automatically updated by the loadgen. Otherwise it is a bug to be fixed in the loadgen.

Reply:
> The comparison should be between the value from the log and the value stored in submission checker right?

Yes, we are currently doing that. But `min_query_count` is a parameter from the mlperf.conf, (a lower limit for the number of queries), not the actual number of queries. The min duration also determines the number of queries.

> @pgmpablo157321 min_query_count is coming from the [loadgen log](https://github.com/pgmpablo157321/inference/blob/min_query_count_check/tools/submission/submission_checker.py#L1121). So, if equal issue mode is enabled, this key value should be automatically updated by the loadgen. Otherwise it is a bug to be fixed in the loadgen.

This is unfortunately also true for the number of queries, we discuss that in the LLM chat and it's a bug that needs to be fixed for v5.0

Reply:
@pgmpablo157321 The check for the equal-issue mode misses commas between benchmark names: [L810-L811](https://github.com/mlcommons/inference/blob/cd3e63e51833abc4eba7241fd0b87f1b5e3b7516/tools/submission/submission_checker.py#L810-L811) and [L812-L813](https://github.com/mlcommons/inference/blob/cd3e63e51833abc4eba7241fd0b87f1b5e3b7516/tools/submission/submission_checker.py#L812-L813)